relayer/pkg/starknet/utils.go

```diff
@@ -46,8 +46,8 @@ func Min[T constraints.Ordered](a, b T) T {
 	return b
 }
 
-// Encodes a byte slice as a bunch of felts. First felt indicates the total byte size.
-func EncodeBytes(data []byte) (felts []*big.Int) {
+// EncodeFelts takes a byte slice and splits as bunch of felts. First felt indicates the total byte size.
+func EncodeFelts(data []byte) (felts []*big.Int) {
```

Comment:
Nit: this makes it seem like we're encoding felts, where we're actually encoding bytes as felts